packages/block-library/src/post-title/edit.js

```diff
@@ -20,5 +42,32 @@ export default function PostTitleEdit( { context } ) {
 		return null;
 	}
 
-	return <h2>{ post.title }</h2>;
+	return (
+		<>
+			<BlockControls>
+				<ToolbarGroup>
+					<HeadingLevelDropdown
+						selectedLevel={ level }
+						onChange={ ( newLevel ) =>
+							setAttributes( { level: newLevel } )
+						}
+					/>
+				</ToolbarGroup>
+				<AlignmentToolbar
+					value={ align }
+					onChange={ ( nextAlign ) => {
+						setAttributes( { align: nextAlign } );
+					} }
+				/>
+			</BlockControls>
```

Comment:
Unrelated to this PR.

@diegohaz @ItsJonQ 

We need to document the move from `ToolbarGroup` to `Toolbar` and what each Toolbar component does. Adding things to the Toolbar is one of the main parts of developing blocks, and right now, it's even hard for Core developers to figure out what the latest way of doing it is.